### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by realbaycan**
223 words were translated.

### CODE REVIEW

**Comment by realbaycan** [comment]
Kolay gelsin Gökay  Bey çevirimi kontrol edebilir misiniz?

**Comment by realbaycan** [comment]
Çevirimi kontrol  eder misiniz?

File: README.asc [hunk 1]
```diff
@@ -1,21 +1,21 @@
-= Pro Git, Second Edition
+= Pro Git, İkinci Baskı
 
-Welcome to the second edition of the Pro Git book.
+Pro Git kitabının ikinci baskısına hoş geldiniz.
 
-You can find this book online at: http://git-scm.com/book
+Bu kitabı çevrimiçi olarak şu adresten bulabilirsiniz: http://git-scm.com/book
```
Comment (jnavila) [inline README.asc]: For turkish, it's https://git-scm.com/book/tr

**Comment by realbaycan** [comment]
correction was made.

**Comment by realbaycan** [comment]
İs it Okey? :)

**Comment by jnavila** [comment]
For me, it's ok.

**Comment by realbaycan** [comment]
Merged?

**Comment by jnavila** [comment]
I cannot merge something I don't understand.
@gokaygurcan ?

**Comment by realbaycan** [comment]
Who will approve? 
@gokaygurcan ?

**Comment by jnavila** [comment]
As of now, he's the maintainer of the project.

**Comment by realbaycan** [comment]
When will he approve? :)

**Comment by jnavila** [comment]
Unfortunately, I don't know...

**Comment by realbaycan** [comment]
Who will approve the my translate?

**Comment by realbaycan** [comment]
Can you help me?

**Comment by gokaygurcan** [comment]
Please, please please please don't ask the same question over and over. It's not sending another notification, it just puts this PR on top in the notification list. I beg you to wait until I reply and do not, I repeat, DO NOT ask the same question over and over. 

PS: I regularly check my notifications and email, there's no way that I can miss any of them, don't worry.

**Pull request closed without merging**